app/elements/stack-blitz.html

```diff
@@ -0,0 +1,186 @@
+<link rel="import" href="./stack-blitz-sdk.html">
```

Comment:
stack-blitz-sdk.html doesn't need to exist (not used elsewhere so doesn't need to be de-duped) - just have `<script src="../js/sdk.umd.js"></script>` here